packages/expo-av/ios/EXAV/AudioSampleCallback/EXAudioSampleCallback.mm

```diff
@@ -0,0 +1,26 @@
+// Copyright 2017-present 650 Industries. All rights reserved.
+
+#import <memory>
+#import <EXAV/EXAudioSampleCallback.h>
+
+using AudioSampleCallbackWrapper = expo::av::AudioSampleCallbackWrapper;
+
+@implementation EXAudioSampleCallback
+{
+  std::unique_ptr<AudioSampleCallbackWrapper> _wrapper;
+}
+
+- (id)initWithCallbackWrapper:(std::unique_ptr<AudioSampleCallbackWrapper>)wrapper {
```

Comment:
```suggestion
- (id)initWithCallbackWrapper:(std::unique_ptr<AudioSampleCallbackWrapper> &&)wrapper {
```

does not really matter with unique ptr but you should probably use rvalue here
note: I don't know if above suggestion is correct syntax for rvalue in objc++